constants/constants.go

```diff
@@ -51,14 +27,46 @@ var (
 	// MagicDirectives are directives automatically appended to Dockerfiles
 	// when pushing the image. These directives allow the built image to be
 	// 're-used'.
-	MagicDirectives = fmt.Sprintf(`
-COPY --chmod=0755 %[1]s %[2]s
-COPY --chmod=0644 %[3]s %[4]s
+	MagicDirectives = `
+COPY --chmod=0755 .envbuilder.tmp/envbuilder /.envbuilder/bin/envbuilder
+COPY --chmod=0644 .envbuilder.tmp/image /.envbuilder/image
 USER root
 WORKDIR /
-ENTRYPOINT [%[2]q]
-`,
-		".envbuilder.tmp/envbuilder", MagicBinaryLocation,
-		".envbuilder.tmp/image", MagicImage,
-	)
+ENTRYPOINT ["/.envbuilder/bin/envbuilder"]
+`
```

Comment:
review: Hard-coding the values here, as they should never change.

Reply:
Nit: I think we should still use the above `defaultMagicDir` const.
They are ostensibly linked conceptually, and if we changed one without the other it could cause problems right?

Reply:
Updated!